@xen-orchestra/lite/src/libs/xen-api.ts

```diff
@@ -193,17 +194,18 @@ export default class XenApi {
       .filter((type: string) => type !== "message");
   }
 
-  get sessionId() {
-    return this.#sessionId;
-  }
+  getSessionId = () => this.#sessionId;
```

Comment:
Why this change?

Reply:
Due to a previous implementation. I will revert this change because it is no longer necessary.